### ISSUE TITLE
python 3.9 test failure: TestSW.test_sw_put_ftp

### ISSUE THREAD

**Issue by bnavigator** [Juniper/py-junos-eznc#1109]
The call representation changed (context manager):

py-junos-eznc 2.6.0
```
================================================================== test session starts ===================================================================
platform linux -- Python 3.9.2, pytest-6.2.2, py-1.10.0, pluggy-0.13.1 -- /usr/bin/python3.9
cachedir: .pytest_cache
rootdir: /home/abuild/rpmbuild/BUILD/py-junos-eznc-2.6.0, configfile: pytest.ini
plugins: forked-1.3.0
_________________________________________________________________ TestSW.test_sw_put_ftp _________________________________________________________________
self = <tests.unit.utils.test_sw.TestSW testMethod=test_sw_put_ftp>, mock_ftp_put = <MagicMock name='FTP' id='140387646799680'>

    @patch("jnpr.junos.utils.sw.FTP")
    def test_sw_put_ftp(self, mock_ftp_put):
        dev = Device(
            host="1.1.1.1",
            user="rick",
            password="password123",
            mode="telnet",
            port=23,
            gather_facts=False,
        )
        dev.facts = facts
        sw = SW(dev)
        sw.put(package="test.tgz")
>       self.assertTrue(call("test.tgz", "/var/tmp") in mock_ftp_put.mock_calls)
E       AssertionError: False is not true
>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>> PDB post_mortem (IO-capturing turned off) >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
> /usr/lib64/python3.9/unittest/case.py(682)assertTrue()
-> raise self.failureException(msg)
(Pdb) up
> /home/abuild/rpmbuild/BUILD/py-junos-eznc-2.6.0/tests/unit/utils/test_sw.py(164)test_sw_put_ftp()
-> self.assertTrue(call("test.tgz", "/var/tmp") in mock_ftp_put.mock_calls)
(Pdb) p mock_ftp_put.mock_calls
[call(Device(1.1.1.1)),
 call().__enter__(),
 call().__enter__().put('test.tgz', '/var/tmp'),
 call().__exit__(None, None, None)]
(Pdb) call("test.tgz", "/var/tmp")
call('test.tgz', '/var/tmp')
(Pdb) 
```

**Comment by rahkumar651991** [comment]
@bnavigator - In Python3.9 some old python api's are no more supported. Working on it.

**Comment by bnavigator** [comment]
Still an issue with Python 3.10

**Comment by dineshbaburam91** [comment]
Fixed #1170
